.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,84 @@
+---
+name: Bug Report
+description: Report a bug in the project
+labels: ["bug"]
+
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: |
+        Describe the bug that you are experiencing, including what you expected to happen
+    validations:
+      required: true
+
+  - type: textarea
+    id: externaldns-configuration
+    attributes:
+      label: ExternalDNS Configuration
+      description: |
+        Configuration of ExternalDNS that you are using (e.g. Helm Values, Kubernetes manifests, etc)
+      placeholder: |
+        ```yaml
+        Paste your configuation here
+        ```
+    validations:
+      required: true
+
+  - type: textarea
+    id: externaldns-logs
+    attributes:
+      label: ExternalDNS and Webhook Logs
+      description: |
+        Logs (be sure to enable debug logging on ExternalDNS and this Webhook)
+      placeholder: |
+        ```text
+        Paste your ExternalDNS and Webhook logs here
+        ```
+    validations:
+      required: true
```

```suggestion
  - type: textarea
    id: externaldns-logs
    attributes:
      label: ExternalDNS Logs
      description: |
        Logs (be sure to enable debug logging on ExternalDNS)
      placeholder: |
        ```text
        Paste your ExternalDNS logs here
        ```
    validations:
      required: true

  - type: textarea
    id: webhook-logs
    attributes:
      label: Webhook Logs
      description: |
        Logs (be sure to enable debug logging on this Webhook)
      placeholder: |
        ```text
        Paste your Webhook logs here
        ```
    validations:
      required: true
```